src/viewer/viewer.ts

```diff
@@ -931,33 +934,42 @@ export default class Viewer {
     x *= window.devicePixelRatio
     y *= window.devicePixelRatio
 
-    let pid, instance, picker
+    x = Math.max(x - 2, 0)
```

Comment:
I think the 2 should be multiplied by `window.devicePixelRatio`

Reply:
Would that change the size of the buffer that we need to read to too?

Reply:
Right, it would need to be changed. Let's forget about it.